src/main/java/io/strimzi/kafka/quotas/VolumeSource.java

```diff
@@ -118,28 +143,50 @@ private CompletionStage<VolumeUsageResult> onDescribeClusterComplete(Result<Coll
 
     private CompletionStage<VolumeUsageResult> onDescribeClusterSuccess(Collection<Node> nodes) {
         final Set<Integer> allBrokerIds = nodes.stream().map(Node::id).collect(toSet());
+        activeBrokerCount.set(allBrokerIds.size());
         log.debug("Attempting to describe logDirs");
         return toResultStage(admin.describeLogDirs(allBrokerIds).allDescriptions())
-                .thenApply(VolumeSource::onDescribeLogDirComplete);
+                .thenApply(this::onDescribeLogDirComplete);
     }
 
-    private static VolumeUsageResult onDescribeLogDirComplete(Result<Map<Integer, Map<String, LogDirDescription>>> logDirResult) {
+    private VolumeUsageResult onDescribeLogDirComplete(Result<Map<Integer, Map<String, LogDirDescription>>> logDirResult) {
         if (logDirResult.isFailure()) {
             return failure(VolumeSourceObservationStatus.DESCRIBE_LOG_DIR_ERROR, logDirResult.getThrowable());
         } else {
             return onDescribeLogDirSuccess(logDirResult.value);
         }
     }
 
-    private static VolumeUsageResult onDescribeLogDirSuccess(Map<Integer, Map<String, LogDirDescription>> logDirsPerBroker) {
+    private VolumeUsageResult onDescribeLogDirSuccess(Map<Integer, Map<String, LogDirDescription>> logDirsPerBroker) {
         if (log.isDebugEnabled()) {
             log.debug("Successfully described logDirs: " + logDirsPerBroker);
         }
-        return success(logDirsPerBroker.entrySet()
+        List<VolumeUsage> volumeUsages = logDirsPerBroker.entrySet()
                 .stream()
                 .flatMap(VolumeSource::toVolumes)
                 .filter(Objects::nonNull)
-                .collect(Collectors.toUnmodifiableList()));
+                .collect(Collectors.toUnmodifiableList());
+
+        activeLogDirsCount.set(volumeUsages.size());
+
+        volumeUsages.forEach(volumeUsage -> {
+            final LinkedHashMap<String, String> tags = new LinkedHashMap<>(defaultTags);
+            tags.put(REMOTE_BROKER_TAG, volumeUsage.getBrokerId());
+            tags.put(LOG_DIR_TAG, volumeUsage.getLogDir());
+            Metrics.newGauge(metricName(VolumeSource.class,  "consumed_bytes", tags), new Gauge<>() {
```

Comment:
Hmm need to follow up and confirm the performance implications of calling new gauge every time.

Reply:
Looks okay to me, it does:

```
protected final <T extends Metric> T getOrAdd(MetricName name, T metric) {
        Metric existingMetric = (Metric)this.metrics.get(name);
        if (existingMetric == null) {
            ... create new metric
        } else {
            return existingMetric;
        }
    }
```
So should just be a hashmap lookup.

Hmm, maybe a problem there though? Will it reference the volumeUsage from the first call forever? Do `consumed_bytes` and `available_bytes` change over time?

Reply:
Also, a thought about if a broker becomes inactive. Do we want the gauge to report it's last known usage indefinitely, or should be gauge be de-registered when the broker is no longer active.

Reply:
Yeah I think your right the metrics will stay constant.

How would the plug-in know when to remove a metric? I suspect retiring broker ID's is probably rare enough that we can ignore the question as the metric will only last util the next broker restart anyway.

Reply:
Fixed in 5d77fb72f8344545ffb0151f93ebdcd6bc42b718